src/ragger/conftest/base_conftest.py

```diff
@@ -128,20 +135,28 @@ def prepare_speculos_args(root_pytest_dir: Path, firmware: Firmware, display: bo
                 raise ValueError(f"File '{lib_path}' missing. Did you compile for this target?")
             speculos_args.append(f"-l{lib_name}:{lib_path}")
 
+    # Check if custom user seed has been provided through CLI or optional configuration.
+    # CLI user seed has priority over the optional configuration seed.
+    if cli_user_seed:
+        speculos_args += ["--seed", cli_user_seed]
+    elif not len(conf.OPTIONAL.CUSTOM_SEED) == 0:
```

Comment:
```suggestion
    elif conf.OPTIONAL.CUSTOM_SEED:
```
Should work no?

Reply:
@xchapron-ledger I rely on the same syntax that @fbeutin-ledger used for other `conf.OPTIONAL` usage. I think I remember he had some good point to use this one...